### ISSUE TITLE
 .DAE sequence did not have animation in Blender 3d

### ISSUE THREAD

**Issue by software-Explorer** [Enumer8/Anim8orTransl8or#2]
Hello, When I played .DAE sequence in Blender there was no animation. Model was loaded properly but there were no keyframes. I tried opening .DAE sequence in Blender 2.79 and in Blender 3.0 and played animation but there was no movements. I used timeline bar to view animation.  .DAE sequence file of mine  https://drive.google.com/file/d/1VaqGgbteTj4Fu6qJlmeVb1q0ci8ImFp3/view?usp=sharing

Thanks for making Anim8orTransl8or.

**Comment by Enumer8** [comment]
Hi, @software-Explorer,

Would you be willing to share the original an8 file? Then I can confirm if there is any issue parsing it or if it is something I don't support yet (e.g. I don't support morph targets). Otherwise, if you have multiple sequences, make sure you are looking at the dae file that corresponds to that sequence.

Thanks.

**Comment by Enumer8** [comment]
Hmm... Even my example an8's animations don't work in Blender anymore. I get errors like:

ERROR: [BackBase-animation] Found 0 IN tangent values for 64 tangents
ERROR: [BackBase-animation] Found 0 OUT tangent values for 64 tangents
Sax FWL (Warning): Animation curve "BackBase-animation" not valid!


I will have to figure out what that means :).

**Comment by software-Explorer** [comment]
Thanks for replying. .DAE and anim8or file are in this folder https://drive.google.com/drive/folders/1V5JDUxCzqkSZ9dD3YWWStBVzdx1K5ycZ?usp=sharing

Enumer8 this is nice. Your work will help 3d game makers. I prefer to play with anim8or mainly for its simplicity so it was nice to find about Anim8orTransl8or. I am new to 3d game development and have little interest in 3d game making in present and more interest in animation but when I will dive in 3d game making Anim8orTransl8or will be helpful.
For what purposes do you use anim8orTransl8or ?

Have a nice day.

**Comment by Enumer8** [comment]
It looks like release v0.6.0 is working (https://github.com/Enumer8/Anim8orTransl8or/releases/tag/v0.6.0). Please try it. I guess the animation interpolation change is what broke it. I will try to make a new release today and perhaps revert the change that doesn't work.


Anim8or is definitely great as an entry point to learn about 3D models and animations. It's what I started with at first. It's free and very simple. Other tools like Blender have a lot bigger learning curve or are expensive. However, it seems like there is no good way to transition out of Anim8or to more advanced tools (at least without leaving everything you've done so far behind). I noticed I could convert the 3D models I made, but not the animations. That's the problem I was trying to solve. Thanks for asking.

**Comment by Enumer8** [comment]
As a super quick workaround, it looks like it works to open the dae file and find "HERMITE" and replace it with "LINEAR".

**Comment by Enumer8** [comment]
I made release v0.7.2 (https://github.com/Enumer8/Anim8orTransl8or/releases/tag/v0.7.2). It should be fully working again. Let me know if it works.

**Comment by software-Explorer** [comment]
It worked. Yahooooo!

I first found about Blender and felt I should stick to it but so many options made me worry and distracted. but after 3, 4 yrs I found about anim8or and was satisfied working on it.
But I know Blender is valuable and is applicable for many things so I am using and learning it also.
But Anim8or has special place in my heart so I always give it first trophy and Blender a second trophy.

Have a nice day.
